labels/physics.toml

```diff
@@ -77,3 +77,8 @@ description = "Maintenance and upkeep improvements"
 color = "FEF2C0"
 name = "🖱️ DX"
 description = "Improvements to the Developer Experience"
+
+["💬 Meeting"]
+color = "BFD4F2"
+name = "💬 Meeting"
+description = "Weekly group meeting"
```

Comment:
So I wouldn't add this to the existing label sets. We don't want Meeting in AmpForm for instance. Better would be a copy of `physics.toml` and then add this Meeting.

Don't forget to to add a new step to `.github/workflows/sync-labels.yml` for this new TOML file